file_transfer_agent.go

```diff
@@ -124,6 +127,7 @@ type snowflakeFileTransferAgent struct {
 	useAccelerateEndpoint       bool
 	presignedURLs               []string
 	options                     *SnowflakeFileTransferOptions
+	streamBuffer                **bytes.Buffer
```

Comment:
As mentioned above - why do we need double pointer here?

Reply:
Changed to `io.Writer`